File: app/src/main/java/com/voiceassist/lixinyu/voiceassist/common/widget/LoadingDialog.java
package com.voiceassist.lixinyu.voiceassist.common.widget;

import android.app.Dialog;
import android.content.Context;
import android.widget.TextView;

import com.voiceassist.lixinyu.voiceassist.R;


public class LoadingDialog extends Dialog {
	
	private TextView tvMessage;

	public LoadingDialog(Context context) {
		super(context, R.style.loading_dialog_style);
		this.setContentView(R.layout.loading_dialog_layout);
		this.setCanceledOnTouchOutside(false);
		
		tvMessage = (TextView) findViewById(R.id.loading_dialog_textview);
	}
	
	public void setText(CharSequence str){
		tvMessage.setText(str);
	}
	
	public void setText(int res){
		tvMessage.setText(res);
	}
   
}

File: app/src/main/java/com/voiceassist/lixinyu/voiceassist/common/widget/LoadingDialog.kt
package com.voiceassist.lixinyu.voiceassist.common.widget

import android.app.Dialog
import android.content.Context
import android.view.View
import android.widget.TextView

import com.voiceassist.lixinyu.voiceassist.R


class LoadingDialog(context: Context) : Dialog(context, R.style.loading_dialog_style) {

    private val tvMessage: TextView

    init {
        this.setContentView(R.layout.loading_dialog_layout)
        this.setCanceledOnTouchOutside(false)

        tvMessage = findViewById(R.id.loading_dialog_textview)
    }

    fun setText(str: CharSequence) {
        tvMessage.text = str
    }

    fun setText(res: Int) {
        tvMessage.setText(res)
    }

}
